src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -2051,17 +2071,18 @@ private Map<String, PendingImportObject<BrAPIListDetails>> mapPendingObsDatasetB
      * @return A map of observation variable dataset objects indexed by dataset name.
      * @throws InternalServerException If the existing dataset summary is not retrieved from the BrAPI server, or an error occurs during API communication.
      */
+    // TODO: add dataset name or id parameter? This only supports top level for now.
```

Comment:
So will this need to be updated to support importing sub observations? If so, is it in a card?

Reply:
I don't think we've finished refinement for that workflow. The implementer may or may not update that method, I added the comment as a hint/suggestion.